README.md

```diff
@@ -132,6 +132,21 @@ architectures:
 - `columnar/dbc:latest`
 - `columnar/dbc:{major}.{minor}.{patch}`, e.g. `columnar/dbc:0.0.1`
 
+## Homebrew
+
+`dbc` is available via the Columnar Homebrew tap. The tap will first need
+to be added to your environment:
+
+```console
```

Comment:
Maybe `sh` instead of `console`?

Reply:
Thanks, I see your point.
But I suspect that the README is not the part of Mkdocs site, is it?